Detect existing RFCs

GaryGSC opened the pull request:
Closes #15. Hopefully. You can read all about my thought process over there.

I haven't actually seen what happens when we try ending a standard change for the second time with [`github-action-end-standard-change`](https://github.com/byu-oit/github-action-end-standard-change), so we shouldn't release this to `v1` until we're comfortable with that interaction.

GaryGSC committed:
Detect existing RFCs

GaryGSC commented on index.js:
```diff
@@ -35,19 +35,35 @@ async function run () {
+Last updated on: ${alreadyCreatedRfc.sys_updated_on}`)
+      // Set outputs for GitHub Actions
+      setOutput('change-sys-id', alreadyCreatedRfc.sys_id)
+      setOutput('work-start', convertServicenowTimestampFromMountainToUtc(alreadyCreatedRfc.work_start))
```
I'm having a hard time telling if we need `convertServicenowTimestampFromMountainToUtc()` in this case, due to some existing bad data (#29, which was recently fixed). I actually suspect that we _don't_ need it here.

stuft2 replied:
I'm not familiar enough on the reasoning for this function to give a recommendation

GaryGSC commented:
Can I get some eyes on this so that we can merge it into `staging` and kick the tires a little?

GaryGSC commented:
@blakemorgan, @stuft2, this should hopefully make you two happy. 😄

GaryGSC commented:
Eh. I guess I can try it from here (`uses: byu-oit/github-action-start-standard-change@detect-existing-rfcs`)

stuft2 reviewed (dismissed):
I'm interested in seeing how this behaves. I'm unfamiliar with the descriptionLike query param but the code looks clean and correct to me.

GaryGSC commented:
I found [some docs for `sysparm_query`](https://developer.servicenow.com/dev.do#!/reference/api/orlando/rest/c_TableAPI). So, that beats divining it from the URL in ServiceNow, as I did originally. Unfortunately, I can't deep-link to the spot where it talks about "LIKE", so you'll need to <kbd>CTRL</kbd> + <kbd>F</kbd> for it.

GaryGSC committed:
Stop converting timezone for `work-start` from Table API

It's already in UTC (unlike the API response when we create a standard change).
